packages/tools/src/tools/displayTools/Labelmap/labelmapDisplay.ts

```diff
@@ -314,15 +314,32 @@ function _setLabelmapColorAndOpacity(
   // @ts-ignore
   actor.getProperty().setInterpolationTypeToNearest();
 
-  if (utilities.actorIsA(actorEntry, 'vtkVolume')) {
-    // @ts-ignore
-    actor.getProperty().setUseLabelOutline(renderOutline);
-    // @ts-ignore
-    actor.getProperty().setLabelOutlineOpacity(outlineOpacity);
-    // @ts-ignore
-    actor.getProperty().setLabelOutlineThickness(outlineWidth);
+  // @ts-ignore
+  actor.getProperty().setUseLabelOutline(renderOutline);
+  // @ts-ignore
+  actor.getProperty().setLabelOutlineOpacity(outlineOpacity);
+
+  const { activeSegmentIndex } = SegmentationState.getSegmentation(
+    segmentationRepresentation.segmentationId
+  );
+
+  // create an array that contains all the segment indices and for the active
+  // segment index, use the activeSegmentOutlineWidth, otherwise use the
+  // outlineWidth
+  // Pre-allocate the array with the required size to avoid dynamic resizing.
+  const outlineWidths = new Array(numColors - 1);
```

Comment:
Would really prefer to just have this map 1-1 with the colours - that means you get an empty spot at the beginning, but I think that is preferable to having to map an off by one, and all the consequent issues you get with that.